src/styles/variables.scss

```diff
@@ -0,0 +1,10 @@
+$block-size: 300px;
```

Comment:
minor thing but since this value means height it may be better to name it $block-height